lesson2/job1/main.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
+
+from flask import Flask
+from flask import typing as flask_typing
+from flask import request as flask_request
+import api
+
+app = Flask(__name__)
+
+
+def is_blank(string):
```

Comment:
IMHO this function looks pretty useless. 
For example, for the blank string `""` it returns True, but for any empty JSON object it returns False (e.g. `{}` or `[]`)

Reply:
Agree, however I use this function to make code look clearer. Also, it could be optimized for JSON too.